samples/projection-management/Program.cs

```diff
@@ -0,0 +1,312 @@
+			settings.ConnectionName = "Projection management client";
+			settings.DefaultCredentials = new UserCredentials("admin", "changeit");
+			return new EventStoreProjectionManagementClient(settings);
+		}
+		#endregion createClient
+
+		private static async Task RestartSubSystem(EventStoreProjectionManagementClient managementClient) {
+		#region RestartSubSystem
+			await managementClient.RestartSubsystemAsync();
+		#endregion RestartSubSystem
+		}
+
+		private static async Task Disable(EventStoreProjectionManagementClient managementClient) {
+
+			#region Disable
+			await managementClient.DisableAsync("$by_category");
+
+			try {
+				await managementClient.DisableAsync("projection that does not exists");
+			} catch (InvalidOperationException e) when (e.Message.Contains("NotFound")) {
+				Console.WriteLine(e.Message);
+
+			}
+			#endregion Disable
+		}
+
+
+		private static async Task Enable(EventStoreProjectionManagementClient managementClient) {
+
+			#region Enable
+			await managementClient.EnableAsync("$by_category");
+			
+			try {
+				await managementClient.DisableAsync("projection that does not exists");
+			} catch (InvalidOperationException e) when (e.Message.Contains("NotFound")) {
+				Console.WriteLine(e.Message);
+			}
+			#endregion Enable
+		}
+
+
+
+
+
+		private static Task Delete(EventStoreProjectionManagementClient managementClient) {
+			// not available 
+			return Task.CompletedTask;
+		}
+
+		private static async Task Abort(EventStoreProjectionManagementClient managementClient) {
+			
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Abort", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				// ignore 
+			}
+			
+			
+			#region Abort
+			// The .net clients prior to version 21.6 had an incorrect behavior: they will save the checkpoint.
+			await managementClient.AbortAsync("countEvents_Abort");
+			// It is safe to call abort multiple times 
+			await managementClient.AbortAsync("countEvents_Abort");
+			#endregion Abort
+
+		}
+
+		private static async Task Reset(EventStoreProjectionManagementClient managementClient) {
+
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Reset", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				// ignore 
+			}
+			
+			#region Reset
+			// Checkpoint will be written prior to resetting the projection
+			await managementClient.ResetAsync("countEvents_Reset");	
+			// It is safe to call abort multiple times 
+			await managementClient.ResetAsync("countEvents_Reset");
+			#endregion Reset
+
+		}
+
+		private static async Task CreateOneTime(EventStoreProjectionManagementClient managementClient) {
```

Comment:
**FIX:** Based on [the RFC](https://github.com/EventStore/home/blob/master/RFCs/021%20-%20gRPC%20Projections.md#decisions) only continuous projections should exist. This [was already applied in the Rust client](https://github.com/EventStore/EventStoreDB-Client-Rust/pull/60).

From the RFC:
> 1. Remove Transient and OneTime projection Creation and List methods from the proto used by the clients. The server should still handle these operations as it does now to preserve backwards compatibility.
> 3. Change the name of CreateContinuousProjection and ListContinuous to CreateProjection and List.

I think that we should document the desired state and use the accepted naming.

Reply:
it's not imported  int the documentation , but added here for completeness as the RFC is not implemented yet

Reply:
The current idea is to have documentation aligned across the clients. So if we have the accepted RFC and some clients already applied it, then, in my opinion, we should only document the final result. Especially, that if we keep the old split, then Rust won't be able to document the snippets properly. If the clients are not ready, then maybe we should wait until they're aligned with RFC.

Reply:
>FIX: Based on the RFC only continuous projections should exist. This was already applied in the Rust client.
it's not tagged as release.

Reply:
then we can wait a long time ?

Reply:
That's more a question to @hayley-jean, @jageall how long will it last.

As I wrote above:
>I suggest skipping the obsolete stuff and use the new RFC nomenclature.
Then we'll document only the stuff that we expect to be available in the long term. Having that we won't suggest people using functionalities that will be removed.

At least that's my perspective, would be good to know @mat-mcloughlin or @alexeyzimarev point of view on that.

Reply:
ok my POV is 
Given 
1/ it's not in the DOC site
2/ it exists in the client v20.x & v21.x
3/ we don't (yet)  have  a documentation per version for the gRPC clients,
Then 
we should at least have some sample usage

Reply:
My thoughts are that, while we don't want to encourage people to use features that will be deprecated at some point, it'd  be a bad experience for there to be no documentation about something that's already been released. We're also focusing on the streams API first, so it may be a while before the clients align on projections.

We'll  need to deprecate those methods before the next release of the client, so the samples could be updated to mention the deprecation when that happens, and then removed when it's removed from the API.

Reply:
We don't have yet any versioning for the gRPC clients, plus currently, each client has its own set of methods implemented with its own assumptions. For example, Java has only the basic projection API provided by an external contributor, and NodeJS has different conventions than .NET and Rust (especially that Rust already implemented RFC, and once it's released, there won't be any obsolete methods.
I'm unsure how we can provide a consistent experience of the gRPC docs in that regard.

It may be worth revisiting and coordinating the priorities of the Engineering and DevAdvocacy teams.

Reply:
this sample is about the .net client.
inconsistencies between client have been documented